johnsonandjohnson/Bodiless-JS
# feat(cli): Bodiless starter

## Changes
- Add bodiless new command
- Rename @bodiless/__cxstarter__ to --cxstarter--

## Test Instructions
- Clone this branch
- Run setup
- switch to packages/bodiless-cli
- `npm run bl new -- --clone-local -i`
- Choose HEAD revision
- Try with both __minimal__ and __cxstarter__ options.
- verify both npm run start and npm run build/serve on both sites
- - for cxstarter - smoke test all styleguide pages.

## Related Issues
Closes #1520
